src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -182,4 +204,63 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
     getSelected() {
         return DomAccess.querySelectorAll(this.el, `${this.options.checkboxSelector}:checked`, false);
     }
+
+    refreshDisabledState(filter) {
```

Comment:
It looks like the category filter now is actually broken.

When I trigger an empty search, only filters on the first category level are active. Others are disabled:

![image](https://user-images.githubusercontent.com/20555112/106872871-1963c180-66d4-11eb-8d3a-a8094450f612.png)

As soon as I select a filter, all filters are being disabled:

![image](https://user-images.githubusercontent.com/20555112/106873062-5760e580-66d4-11eb-8b76-219c32609e67.png)

The selected filters looks odd:

![image](https://user-images.githubusercontent.com/20555112/106873208-837c6680-66d4-11eb-9c05-99fe1b7fd5ea.png)

At the same time the filter isn't even selected and the URL looks like this: http://localhost:8000/search?cat=Outdoors%7COutdoors&order=score&p=1&search=

Reply:
@TheKeymaster 
> As soon as I select a filter, all filters are being disabled:

This was how it was originally implemented - and also was a requirement at that time, because we can only select one category filter. This is not related to the disabling filter change. 

> When I trigger an empty search, only filters on the first category level are active. Others are disabled:

This happens because we are only getting the main category from the response. Might need to change the response array so the children are also available in the array.

Reply:
As discussed, will be fixed in SW-578.